fakepkg

```diff
@@ -55,19 +56,15 @@ function fakebuild() {
 
 	# Assembling a list of all files belonging to the package
 	# The tar command is unable to handle very many arguments therefore create a file
-	pacman -Qlq $1 > ./package_file_list
-	echo ".PKGINFO" >> ./package_file_list
-	[[ -f ".INSTALL" ]] && echo ".INSTALL" >> ./package_file_list
-	[[ -f ".Changelog" ]] && echo ".Changelog" >> ./package_file_list
-	[[ -f ".MTREE" ]] && echo ".MTREE" >> ./package_file_list
+	pacman --dbpath "${ROOT_DIR}/var/lib/pacman" -Qlq $1 | sed 's%^/%%' > ./package_file_list
```

Comment:
Why not use the `--sysroot` option here instead as it would save us from having to hard-code the DB path?
At least make it some variable and change line 15 as well as to make the path somewhat configurable.

Reply:
`--sysroot` does look compelling. OTOH it does require root privileges which `--dbpath` approach might not need.

Reply:
You are right, requiring root would not be ideal. Just stick to specifying `--dpath`.

Regardless of the used option, may I ask you to put `/var/lib/pacman/` into a variable and use it throughout the script.